auth/src/user.rs

```diff
@@ -206,7 +235,7 @@ impl User {
         Ok(false)
     }
 
-    // Synchronize the tiers with the subscription validity.
+    /// Synchronize the tiers with the subscription validity.
```

Comment:
nit: Why three slashes? Should this function be considered public-facing, needing doc comments? (https://doc.rust-lang.org/reference/comments.html#doc-comments_)

Reply:
I think doc comments are always useful, for example in VsCode the comment shows on hover, so we can quickly see the comment wherever this function is used.

Reply:
Ah, I see. sounds good then